src/UserConfig.elm

```diff
@@ -1651,6 +1651,411 @@ Podemos cerrar su cuenta de inmediato si pensamos que ha:
+
+
+### 😔 วิธีการยื่นเรื่องร้องเรียน
+
+หากคุณมีเรื่องร้องเรียน กรุณาติดต่อเราโดยส่งอีเมลถึง [""" ++ Env.contactEmailAddress ++ """](mailto:""" ++ Env.contactEmailAddress ++ """) และเราจะทำทุกทางเพื่อแก้ไขปัญหา
+
+
+### 📝 การเปลี่ยนแปลงนโยบายนี้
+
+เราจะโพสต์การเปลี่ยนแปลงที่เราทำเกี่ยวกับแจ้งเตือนความเป็นส่วนตัวของเราในหน้านี้และถ้ามีการเปลี่ยนแปลงที่สำคัญเราจะแจ้งให้คุณทราบทางอีเมล
+
+"""
+    , privacyNotice = "แจ้งเตือนความเป็นส่วนตัว"
+    , profile = "โปรไฟล์"
+    , readMore = "อ่านเพิ่มเติม"
+    , recancelEvent = "ยกเลิกกิจกรรมอีกครั้ง"
+    , reset = "รีเซ็ต"
+    , save = "บันทึก"
+    , saveChanges = "บันทึกการเปลี่ยนแปลง"
+    , saving = "กำลังบันทึก..."
+    , search = "ค้นหา"
+    , searchForGroups = "ค้นหากลุ่ม"
+    , subscribingToOne = "กำลังสมัครสมาชิก"
+    , searchResultsFor = "ผลการค้นหาสำหรับ "
+    , showAll = "แสดงทั้งหมด"
+    , showFirst = "แสดงครั้งแรก"
+    , showAttendees = "(แสดง\u{00A0}ผู้เข้าร่วม)"
+    , signInAndWeWillGetYouSignedUpForThatEvent = "ลงชื่อเข้าใช้และเราจะลงทะเบียนให้คุณเข้าร่วมกิจกรรมนั้น"
+    , signInAndWeWillGetYouSignedUpForThe = \eventName -> "ลงชื่อเข้าใช้และเราจะลงทะเบียนให้คุณเข้าร่วม " ++ eventName ++ " กิจกรรม."
+    , sinceThisIsYourFirstGroupWeRecommendYouReadThe = "เนื่องจากนี่เป็นกลุ่มแรกของคุณ เราขอแนะนำให้คุณอ่าน "
+    , sorryThatGroupNameIsAlreadyBeingUsed = "ขออภัย ชื่อกลุ่มนี้ถูกใช้งานแล้ว"
+    , stopNotifyingMeOfNewEvents = "หยุดแจ้งเตือนเหตุการณ์ใหม่"
+    , submit = "ส่ง"
+    , subscribedGroups = "กลุ่มที่สมัครสมาชิก"
+    , terms = "เงื่อนไข"
+    , theEventCanTStartInThePast = "กิจกรรมไม่สามารถเริ่มในอดีต"
+    , theEventIsTakingPlaceNowAt = "• กิจกรรมกำลังจัดขึ้นในขณะนี้ที่ "
+    , theEventWillTakePlaceAt = "• กิจกรรมจะจัดขึ้นที่ "
+    , theLinkYouUsedIsEitherInvalidOrHasExpired = "ลิงก์ที่คุณใช้ไม่ถูกต้องหรือหมดอายุแล้ว"
+    , theMostImportantRuleIs = "กฎที่สำคัญที่สุดคือ"
+    , theStartTimeCanTBeChangedSinceTheEventHasAlreadyStarted = "ไม่สามารถเปลี่ยนเวลาเริ่มต้นเนื่องจากกิจกรรมเริ่มแล้ว"
+    , thisEventDoesNotExist = "กิจกรรมนี้ไม่มีอยู่"
+    , thisEventSomehowDoesNotExistTryRefreshingThePage = "กิจกรรมนี้ไม่มีอยู่ ลองรีเฟรชหน้า?"
+    , thisEventWasCancelled = "กิจกรรมนี้ถูกยกเลิก "
+    , thisEventWillBeInPerson = "กิจกรรมนี้จะจัดขึ้นโดยตรง"
+    , thisEventWillBeOnline = "กิจกรรมนี้จะจัดขึ้นออนไลน์"
+    , thisEventWillBeOnlineAndInPerson = "กิจกรรมนี้จะจัดขึ้นออนไลน์และโดยตรง"
+    , thisGroupHasTooManyEvents = "กลุ่มนี้มีกิจกรรมมากเกินไป"
+    , thisGroupWasCreatedOn = "กลุ่มนี้ถูกสร้างขึ้นเมื่อ "
+    , timeDiffToString = diffToStringEnglish
+    , timeValueMissing = "ไม่มีค่าเวลา"
+    , title = "กิจกรรม"
+    , tos = "ข้อกำหนดในการให้บริการ"
+    , tosMarkdown = \privacyRoute codeOfConductRoute -> """
+
+#### รุ่น 1.0 - มิถุนายน 2021
+
+### 🤔 Meetdown คืออะไร
+
+เงื่อนไขกฎหมายเหล่านี้คือสิ่งที่คุณและ meetdown.app (“เรา”, “ของเรา”, “Meetdown”, ซอฟต์แวร์) ตกลงกัน และคุณตกลงตามเงื่อนไขเหล่านี้โดยการใช้บริการ Meetdown
```

Comment:
i can get the gist but this flows weirdly

Reply:
_Key: `tosMarkdown`_

**Original**: These legal terms are between you and meetdown.app (“we”, “our”, “us”, “Meetdown”, the software”) and you agree to them by using the Meetdown service.

**Old Translation**: เงื่อนไขกฎหมายเหล่านี้คือสิ่งที่คุณและ meetdown.app (“เรา”, “ของเรา”, “Meetdown”,ซอฟต์แวร์) ตกลงกัน และคุณตกลงตามเงื่อนไขเหล่านี้โดยการใช้บริการ Meetdown

_Dtinth Review: Probably mistranslated_

**New Translation**: เงื่อนไขกฎหมายเหล่านี้เป็นสิ่งที่คุณ

Reply:
```suggestion
เงื่อนไขกฎหมายเหล่านี้เป็นสิ่งที่คุณและ meetdown.app ("เรา", "ของเรา", "Meetdown", ซอฟต์แวร์) ตกลงกัน คุณยอมรับเงื่อนไขเหล่านี้โดยการใช้บริการ Meetdown
```